### ISSUE TITLE
Rebuilding snapshots fails with ENAMETOOLONG error

### ISSUE THREAD

**Issue by jaydenseric** [tapjs/tapjs#567]
It seems there is a bug where  if your snapshots are in a lot of separate test files, each time snapshots are rebuilt a huge nonsensical filename is passed to `rimraf` resulting in an `ENAMETOOLONG: name too long` error.

For example, with the following snapshot files:

<img width="295" alt="Screen Shot 2019-06-05 at 6 21 46 pm" src="https://user-images.githubusercontent.com/1754873/58941491-405f4700-87bf-11e9-82b4-e9bdace92e4a.png">

Here is the CLI error after running `tap lib/*.test.js`:

```
/Users/jaydenseric/Sites/jsdoc-md/node_modules/yoga-layout-prebuilt/yoga-layout/build/Release/nbind.js:53
        throw ex;
        ^

Error: ENAMETOOLONG: name too long, unlink '/Users/jaydenseric/Sites/jsdoc-md/tap-snapshots/node_modules-.bin-tap-lib-deconstructJsdocNamepath.test.js-lib-getJsdocAstTag.test.js-lib-getJsdocAstTags.test.js-lib-jsdocCommentsFromCode.test.js-lib-jsdocMd.test.js-lib-jsdocToMember.test.js-lib-mdFileReplaceSection.test.js-lib-mdToMdAst.test.js-lib-membersToMdAst.test.js-lib-outlineMembers.test.js-lib-remarkPluginReplaceSection.test.js-lib-replaceJsdocLinks.test.js-lib-typeJsdocAstToMdAst.test.js-TAP.test.js'
    at Object.unlinkSync (fs.js:956:3)
    at Function.rimrafSync [as sync] (/Users/jaydenseric/Sites/jsdoc-md/node_modules/rimraf/rimraf.js:306:17)
    at Snapshot.save (/Users/jaydenseric/Sites/jsdoc-md/node_modules/tap/lib/snapshot.js:56:14)
    at TAP.process (/Users/jaydenseric/Sites/jsdoc-md/node_modules/tap/lib/test.js:411:27)
    at TAP.onbufferedend (/Users/jaydenseric/Sites/jsdoc-md/node_modules/tap/lib/test.js:568:10)
    at Spawn.p.runMain [as cb] (/Users/jaydenseric/Sites/jsdoc-md/node_modules/tap/lib/test.js:463:32)
    at Spawn.callCb (/Users/jaydenseric/Sites/jsdoc-md/node_modules/tap/lib/spawn.js:103:12)
    at Spawn.onprocclose (/Users/jaydenseric/Sites/jsdoc-md/node_modules/tap/lib/spawn.js:135:17)
    at ChildProcess.proc.on (/Users/jaydenseric/Sites/jsdoc-md/node_modules/tap/lib/spawn.js:90:47)
    at ChildProcess.emit (events.js:198:13)
```

Environment:

- Node.js v10.16.0
- tap v14.2.1

**Comment by isaacs** [comment]
Wow, that's unusual. I'll take a look tomorrow.

**Comment by isaacs** [comment]
Fixed on 14.2.2.

Note: you don't usually have to specify the test file arguments on the command line any more (which is probably why this error slipped past my notice).  Just running `tap` will find all the `*.test.js` files.
